timely/src/progress/subgraph.rs

```diff
@@ -636,7 +641,8 @@ impl<T: Timestamp> PerOperatorState<T> {
         mut scope: Box<dyn Operate<T>>,
         index: usize,
         identifier: usize,
-        logging: Option<Logger>
+        logging: Option<Logger>,
+        summary_logging: &mut Option<SummaryLogger<T::Summary>>,
```

Comment:
Alternatively `Option<&mut SummaryLogger<...>>`? I can't seem to ever know which one to use where, but here we don't want to change the option... Also, logging mightn't need mutable access iirc

Reply:
Will address in a follow-up!